module-4/backend/week_three.md

```diff
@@ -5,15 +5,33 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. At a high level, what is Node?
+A tool used to run server-side JS. It is single process. 
 2. What is Express? What is Express similar to in the Ruby world?
+Express is a framework for Node.js. It is similar to ruby in its web application building capabilities. 
```

Comment:
i wouldn't say express and ruby are similar. ruby is a language and express is a framework written in javascript. express is more similar to sinatra